app/Domain/Contacts/Repositories/EloquentContactRepository.php

```diff
@@ -63,4 +63,13 @@ public function getDeletedById(UuidInterface $id): Contact
 
         return $contactModel->toDomain();
     }
+
+    public function getByIntegrationIdAndEmail(UuidInterface $integrationId, string $email): Collection
```

Comment:
Maybe add this method after `getByIntegrationId` and type the return statement.

We used to have these generic return types as a requirement:
```
    checkGenericClassInNonGenericObjectType: false
    checkMissingIterableValueType: false
```

Reply:
I created a ticket to improve the PHPStan settings, see https://jira.publiq.be/browse/PPF-261

Reply:
Done